src/definitions/behaviors/form.js

```diff
@@ -660,13 +660,16 @@ $.fn.form = function(parameters) {
             ;
             $fields.each(function(index, field) {
               var
-                $field     = $(field),
-                name       = $field.prop('name'),
-                value      = $field.val(),
-                isCheckbox = $field.is(selector.checkbox),
-                isRadio    = $field.is(selector.radio),
-                isMultiple = (name.indexOf('[]') !== -1),
-                isChecked  = (isCheckbox)
+                $field       = $(field),
+                $parent      = $field.parent(),
+                $grandParent = $parent.parent(),
+                name         = $field.prop('name'),
+                value        = $field.val(),
+                isCheckbox   = $field.is(selector.checkbox),
+                isRadio      = $field.is(selector.radio),
+                isMultiple   = (name.indexOf('[]') !== -1),
+                isCalendar   = ($parent.is(selector.uiCalendar) || $grandParent.is(selector.uiCalendar)),
```

Comment:
What about using the `.closest()` functionality instead? You could get rid of the additional $parent/$grandparent variables and, in case the DOM structure changes, the code is already prepared prepared:

```javascript
$calendar = $field.closest(selector.uiCalendar),
isCalendar   = ($calendar.length>0),
```

Reply:
Wow good catch here ! I think that I have over complicated the code 👍